js/image/example/Image.svelte

```diff
@@ -1,17 +1,22 @@
 <script lang="ts">
+	import { Image } from "@gradio/wasm/svelte";
+
 	export let value: string;
 	export let samples_dir: string;
 	export let type: "gallery" | "table";
 	export let selected = false;
 </script>
 
 <!-- TODO: fix -->
-<!-- svelte-ignore a11y-missing-attribute -->
-<img
+<Image
 	src={samples_dir + value}
-	class:table={type === "table"}
-	class:gallery={type === "gallery"}
-	class:selected
+	class={[
```

Comment:
The `class:` directive can't be used to components 😵‍💫

Any advice? Sorry I'm not so familiar with Svelte convention.

Reply:
Also these styles defined at the parent component don't work...

Reply:
Yes, svelte scopes styles to ensure component encapsulation.

The best approach here would be not to pass the classes down but to reimplement the styles to apply to a parent span or div, then the image can just have a 100% width/ height and the container can take care of the styling. I don't think we are doing anything too fancy with the example media so this should work well and will avoid the need to pass down the classes.

If we _do_ need to apply style to the image from this component we can use a descendant global selector which will allow us to target the `img` but not leak out styles:

```
css.img-wrap :global(> img) {
 ...
}
```

Reply:
Thanks!